src/common/models/measure/measure.ts

```diff
@@ -224,22 +232,34 @@ export class Measure implements Instance<MeasureValue, MeasureJS> {
     return this.formatFn(datum[this.name] as number);
   }
 
-  public change(propertyName: string, newValue: any): Measure {
+  public change(propertyName: string, newValue: any): this {
```

Comment:
yeah it is meant to indicate that the function return something if it's own type. This is useful when when function can be inherited but still return the type of the object being called.

Hold on I will add more info in a sec.